VideoAPI/VideoApi.IntegrationTests/Helper/TestDataManager.cs

```diff
@@ -34,6 +34,7 @@ public async Task<Conference> SeedConference()
                 .WithConferenceStatus(ConferenceState.InSession)
                 .WithMeetingRoom(_services.PexipNode, _services.ConferenceUsername)
                 .WithHearingTask("Suspended")
+                .WithMessages(2)
```

Comment:
Not every test will need messages. Could you please ensure only conferences that need messages have messages included?

Reply:
Removed it